lib/elixir/src/elixir_module.erl

```diff
@@ -70,16 +70,18 @@ compile(Line, Module, Block, Vars, E) ->
 
   try
     put_compiler_modules([Module | CompilerModules]),
-    {Result, NE} = eval_form(Line, Module, Data, Block, Vars, E),
+    {Result, NE, OverridablePairsNotOverridden} = eval_form(Line, Module, Data, Block, Vars, E),
```

Comment:
I believe we can simply call this `Overridable` or `OverridablePairs`. Something that is overridable by definition was not overridden yet. Once it is overridden, it is no longer overridable.